File: laboratorna 3/src/Ointment.java
public class Ointment extends Goods{
    private int volume;

    public Ointment(String name, double price,  int expirationDate, CategoryType categoryType,int volume) {
        super(name, price, expirationDate, categoryType );
        this.volume = volume;
    }

    public int getVolume() {
        return volume;
    }

    public void setVolume(int volume) {
        this.volume = volume;
    }

    @Override
    public String toString() {
        return "Ointment{" + "name='" + super.getName() + '\'' +
                ", price=" + super.getPrice() +
                ", expirationDate=" + getExpirationDate() +
                ", categoryType=" + getCategoryType() +
                "volume=" + volume +
                '}';
    }
}

File: lab-11 python/Ointment.py
from Goods import Goods

class Ointment(Goods):

    def __init__(self, name, price, expiration_date, category_type, volume):
        super(Ointment, self).__init__(name, price, expiration_date, category_type)
        self.volume = volume


def __str__(self):
    return super(Ointment, self).__str__() + ' volume = ' \
           + str(self.volume)